RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 20.4
 -----
 * [*] Site Creation: Fixed a bug in the design picker where the horizontal position of designs could be reset. [#19020]
+* [*] User Mention: Split the suggestions list into a prominent section and a regular section. [#19064]
```

Comment:
@twstokes This was added last week. But since 20.4 is now frozen, I think this PR will be released in 20.5?

Reply:
Exactly right @salimbraksa. 👍 It looks like `trunk` currently doesn't have a 20.5 section, so feel free to add it in this PR.